### DISCUSSION TITLE
Windows 10 Install

### DISCUSSION THREAD

**Discussion by SwissMontreux** [coqui-ai/TTS#1857] [General Q&A]
Has anyone sucessfully installed TTS on a Windows 10 box?  I'm having nothing but total ballache trying to get it installed.. errors can be provided, but the first error was MS C++ build tools not having the correct version installed, that was fix and now a new set of errors, the new error implies that Cython is not installed, which it is as per the below:-

Package    Version
---------- -------
Cython     0.29.32
numpy      1.23.1
pip        22.2.2
pyworld    0.2.10
setuptools 41.2.0
wheel      0.37.1

**Comment by SwissMontreux** [comment]
As always in life, you spend many hours trying to fix a problem, then post the issue online and you continue looking at the problem.. and you fix it. The Windows install documentation is misleading tbch and the problem was around where pip was installing the modules v running TTS install via .\scripts\pip install e .  There was also the issue of MS C++ missing as well, or the correct version at least. So I now have Windows training a model with an old'ish GPU and its flying!!!

**Reply by amrpyt** [reply]
**so how do you fix it?**

**Reply by thorstenMueller** [reply]
Not sure if it helps you @amrpyt , but i've created a step by step tutorial video on how to do voice cloning with Coqui TTS on Windows 10 - including dependencies.
https://youtu.be/bJjzSo_fOS8
